python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -499,14 +460,22 @@ async def enable_devices(self):
         self.log.info("Enable devices")
         self.disable_task.cancel()
         try:
-            enable_commands = [
+            for reset_command in [
+                commands.SafetyReset(what="25"),  # Reset e-stops.
```

Comment:
Should "25" be an attribute or something like that? Is there a list of all the possible "what" and their meaning?

I think something like:

```
commands.SafetyReset(what=SafetyCode.ESTOP)
```

Reply:
The code would have to be:

`what=f"{SafetyCode.ESTOP}"`

I agree it'd be useful, though I wish the "what" was a bitmask. In any case I only have the codes for the CCW and I think they are going to change for the full TMA so I suggest holding off on this until we have the final information.

Reply:
Looking through the latest discussion of e-stop reset this is going to disappear in the full (TMA) version. I'll add a code comment and leave it at that.